### PR TITLE
cut 0.15.0

### PR DESCRIPTION
**Pull request by Nexucis**
Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+0.15.0 / 2021-04-13
+===================
+
+* **[Feature]**: Provide a way to inject an initial metric list for the autocompletion (#134)
+* **[Enhancement]**: Autocomplete metrics/function/aggregation when the editor is empty (#133)
+* **[Enhancement]**: Improve the documentation to reflect what the lib is providing. (#134)
+* **[Change]**: Export the essential interface in the root index of the lib. (#132)
+* **[Change]**: Downgrade the NodeJS version required (from 14 to 12) (#112)  
+* **[BreakingChange]**: Support CommonJS module. (#130) 
+  Be careful it changes the import path if you are using something not
```
Comment (juliusv) [inline CHANGELOG.md:10]: I'd add an empty line between this paragraph and the last bullet point above.

Comment (juliusv) [inline CHANGELOG.md:10]: "Be careful it changes" -> I would change this to something like "Note that this release requires changes to your import paths if..."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+  exported by the root index of lib. For example: `import { labelMatchersToString } from 'codemirror-promql/parser/matcher';`
+  become `import { labelMatchersToString } from 'codemirror-promql/esm/parser/matcher';`
```
Comment (juliusv) [inline CHANGELOG.md:12]:
```suggestion
  becomes `import { labelMatchersToString } from 'codemirror-promql/esm/parser/matcher';`
```

**Comment by juliusv** [comment]
:+1: besides comments

**Comment by Nexucis** [comment]
thanks for the review @juliusv :)

**Pull request merged**